src/VisualStudio/Setup/source.extension.vsixmanifest

```diff
@@ -48,5 +48,6 @@
   </Assets>
   <Prerequisites>
     <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
+    <Prerequisite Id="Microsoft.DiaSymReader.Native" Version="[15.0,16.0)" DisplayName="Windows PDB reader/writer" />
```

Comment:
Do we need the same entry in 

https://github.com/dotnet/roslyn/blob/master/src/Compilers/Extension/source.extension.vsixmanifest